### PR TITLE
Shared element transition in SummaryProductFragment

### PR DESCRIPTION
**Pull request by amanjeetsingh150**
## Description

Added the shared element transition in SummaryProductFragment for products.

## Related issues and discussion
#1277 
 
 ## Screen-shots, if any
<br>
 
 
 ## Checklist

Make sure you've done all the following (You can delete the checklist before submitting)
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 - [x] Code is well documented
 - [x] Include unit tests for new functionality
 - [x] All user-visible strings are made translatable
 - [x] Code passes Travis builds in your branch
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 - [x] Read and understood the contribution guidelines .

### CODE REVIEW

**Comment by amanjeetsingh150** [comment]
@teolemon @Karljoones Please review :)

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -487,4 +487,7 @@
     <string name="connection">Connected</string>
     <string name="contributor_string">Contributor</string>
 
+    <!---->
+    <string name="product_transition">productTransition</string>
```
Comment (Karljoones) [inline app/src/main/res/values/strings.xml]: Should be non-translateable

**Comment by amanjeetsingh150** [comment]
Hey,
@Karljoones @gulzar1996 Thanks for your review. I have updated my PR accordingly. Please review.
Thanks :+1:

**Comment by Karljoones** [comment]
Merged, thanks @amanjeetsingh150

**Pull request merged**